uint8 gACT #getActivation
uint8 gGTO #getActionStatus
uint8 gSTA #getMotionStatus
uint8 gOBJ #getObjectDetection
uint8 gFLT #getFaultStatus
uint8 gPR #getPreviousPositionRequest
uint8 gPO #getPosition
uint8 gCU #getCurrent
